Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -83,6 +83,15 @@ var DrawerLayoutAndroid = React.createClass({
       'on-drag',
     ]),
     /**
+     * Specifies the background color of the drawer. The default value is white.
+     * If you want to set the opacity of the drawer, use rgba. Example:
```

Comment:
Can you test that this displays properly on the website. You can run it locally with
```
cd website
npm install
npm start
```
You can use ``` to format the example code properly.